results/github.com/tektoncd/experimental/results/proto/proto/taskrun.pb.go

```diff
@@ -0,0 +1,1653 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
We should probably delete this file if we can.

1) there's the existing `results/proto/proto/api.pb.go`
2) the import path for this is redundant (e.g. the fully resolved path would be `github.com/tektoncd/experimental/results/results/github.com/tektoncd/experimental/results/proto/proto`)

Reply:
done